### PR TITLE
Support dash in name of language

### PR DESCRIPTION
**Pull request by dg-ratiodata**
PR for #244.

### CODE REVIEW

**Comment by dg-ratiodata** [comment]
This is not finished yet. I was not sure what is needed to run the test suite. I will update the PR, when I understand your development environment.

File: lib/kramdown/parser/kramdown/codeblock.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def parse_codeblock
 
 
       FENCED_CODEBLOCK_START = /^~{3,}/
-      FENCED_CODEBLOCK_MATCH = /^((~){3,})\s*?(\w+)?\s*?\n(.*?)^\1\2*\s*?\n/m
+      FENCED_CODEBLOCK_MATCH = /^((~){3,})\s*?(\w+-)?\s*?\n(.*?)^\1\2*\s*?\n/m
```
Comment (gettalong) [inline lib/kramdown/parser/kramdown/codeblock.rb]: The change would need to be from `(\w+)` to `(\w[\w-]*)`.

Comment (maxmeyer) [inline lib/kramdown/parser/kramdown/codeblock.rb]: Sure. Must have been blind. Fixed that, but forgot to push. :smile: But there's some more code which needs to be changed. Somewhere "you" may split the html-class-name. Only parts of the same go to `class="lang"`

**Comment by gettalong** [comment]
See #246 for the new PR.

**Pull request closed without merging**